scripts/build-probe-binaries

```diff
@@ -38,14 +38,14 @@ fi
 function update_code_for {
 	repo=$1
 	if [ ! -d $repo ]; then
-		git clone git@github.com:draios/$repo.git
+		git clone https://github.com/draios/$repo.git
```

Comment:
Add a comment here stating why this change is required.

Reply:
inside container the git credential (provided in jenkins job) by default is inaccessible. it happens that the repo  we are using here (sysdig) is public accessible, https protocol suffices. In case of private reos, we need do extra to make git credential accessible in container. For module build, it is not required so far.